src/defrag-hash.c

```diff
@@ -359,7 +358,7 @@ typedef struct DefragHashKey4_ {
         struct {
```

Comment:
this struct is now bigger than the `u32` array that is supposed to match it

Reply:
Would adding a `uint16_t pad` work ok with `hashword`?